lstchain/tools/lstchain_create_drs4_pedestal_file.py

```diff
@@ -0,0 +1,112 @@
+"""
+Create drs4 time pedestal fits file.
+"""
+import numpy as np
+from astropy.io import fits
+from tqdm.autonotebook import tqdm
+
+from ctapipe.core import Provenance, traits
+from ctapipe.core import Tool
+from ctapipe.io import EventSource
+from lstchain.calib.camera.drs4 import DragonPedestal
+
+
+class PedestalFITSWriter(Tool):
+
+    name = "PedestalFITSWriter"
+    description = "Generate a pedestal FITS file"
+
+    output = traits.Path(
+        default_value="pedestal.fits",
+        help="Path to the generated fits pedestal file",
+        directory_ok=False,
+    ).tag(config=True)
+
+    deltaT = traits.Bool(
```

Comment:
Sorry to come at this again, but you did the opposite of what I proposed. Now the default is "no delta T correction" and one has to explicitly enable it.

Do we want that? Isn't the normal use case the one *with* delta T correction? Defaults should be as close to the normal usage as possible, so less options are required for normal use, while still enabling the not-normal use.

Reply:
I understand that this is how it works in the [lstchain_data_create_drs4_pedestal_file.py](https://github.com/cta-observatory/cta-lstchain/blob/master/lstchain/scripts/lstchain_data_create_drs4_pedestal_file.py#L58) script.

Reply:
No, the code seems to by default make the calibration and you have to pass `--deltaT=false` to deactivate it.

Reply:
So again, for the new tool I would propose having `deltaT` with default value `True` and an option `--no-deltaT` that sets to false